docs/en/sql-reference/sql-statements/data-definition/ALTER_TABLE.md

```diff
@@ -963,6 +1004,112 @@ The `be_compactions` table in the `information_schema` database records compacti
     MODIFY COLUMN v2 DOUBLE DEFAULT "1" AFTER v1;
     ```
 
+14. Add and drop fields in STRUCT-type data.
+
+    Create a table and insert a row of data.
```

Comment:
with a `-` ?

Reply:
It is not a part of the example. It's the preparation.